### PR TITLE
stop passing boostrap args in config

### PR DESCRIPTION
**Pull request by zshihang**
see https://github.com/kubernetes/test-infra/pull/19353#pullrequestreview-509740356

### CODE REVIEW

File: config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml [hunk 1]
```diff
@@ -14,15 +14,13 @@ periodics:
...
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201014-d8fda77-master
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20201014-882eb3f-master
+        command:
+        - runner.sh
+        - kubetest
         args:
-        - --timeout=920
-        - --bare
-        - --scenario=kubernetes_e2e
-        - --
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml:22]: This is missing flags to tell kubetest what to do, I would add these:
https://github.com/kubernetes/test-infra/blob/deb1bd1036360a6357f5d7948bc287301991c213/config/jobs/kubernetes/sig-cloud-provider/azure/sig-azure-config.yaml#L73-L82

File: config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml [hunk 2]
```diff
@@ -31,3 +29,8 @@ periodics:
         - --provider=gce
         - --upgrade_args="--ginkgo.focus=\[Feature:BoundServiceAccountTokenVolume\] --upgrade-target=ci/latest --upgrade-image=gci"
         - --timeout=60m
+        securityContext:
+          privileged: true
```
Comment (spiffxp) [inline config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml:33]: This should not be needed

**Review by spiffxp — CHANGES_REQUESTED** [review]
Needs some changes

File: config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml (hunk 1 above)
Comment (zshihang) [reply config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml:22]: --upgrade_arg will run the upgrade test.

`kubetest --extract=ci/latest --extract=ci/latest-1.19 --check-version-skew=false --check-leaked-resources --gcp-node-image=gci  --gcp-zone=us-west1-b --upgrade_args="--ginkgo.focus=\[Feature:BoundServiceAccountTokenVolume\] --upgrade-target=ci/latest --upgrade-image=gci"` works for me locally.

File: config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml (hunk 2 above)
Comment (zshihang) [reply config/jobs/kubernetes/sig-auth/serviceaccount-admission-controller-migration-config.yaml:33]: done

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Comment by spiffxp** [comment]
/retest
bazel network/disk flake
```Image pull was unsuccessful: unable to save remote image gcr.io/k8s-testimages/launcher.gcr.io/google/bazel@sha256:2e8163b61f3759f6ff0e4df43c40d092dae331b1c2d5326f05f78e72a68d3203: unable to write image layers: unable to write image layer: unable to write the contents of layer 0 to /bazel-scratch/.cache/bazel/_bazel_root/05618a594cb3499a4b912817865867cd/external/bazel-base/image/000.tar.gz: read tcp 10.60.115.22:51924->108.177.111.128:443: read: connection reset by peer```

**Pull request merged**